src/delphi/vcl.castlecontrol.pas

```diff
@@ -14,7 +14,7 @@
 }
 
 { Control with OpenGL context on a Delphi VCL form. }
-unit Vcl.CastleControl;
+unit vcl.castlecontrol;
```

Comment:
Do not make the unit name lowercase. Please keep it `PascalCase`, so `Vcl.CastleControl`.